### DISCUSSION TITLE
Amasty_Storelocator

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#131] [3. Integrations]
Hello,

The module Amasty_Storelocator - [https://amasty.com/store-locator-for-magento-2.html)](https://amasty.com/store-locator-for-magento-2.html) is not compatible.
Please check.

**Comment by dimdimich** [comment]
Hi, thank you for your using Breeze. Can you please let us know if you are already contacted Amasty about Breeze integration?
